test/kube_testing/rbac/cluster_roles.go

```diff
@@ -114,6 +114,11 @@ func CreateRoleAdmin(name string, namespace string) Role {
 					Resources: []string{"configmaps"},
 					Verbs:     []string{"get"},
 				},
+				{
```

Comment:
That is a good finding. Can you please split it to a separate PR.

Reply:
Oh actually maybe no. I let you decide.

Reply:
Tests on my branch will fail without these changes